<?xml version="1.0" encoding="UTF-8"?>
<module external.linked.project.id=":06._Try_with_resources-What_can_be_released:main" external.linked.project.path="$MODULE_DIR$/../../../../06. Try with resources/What can be released" external.root.project.path="$MODULE_DIR$/../../../.." external.system.id="GRADLE" external.system.module.group="19._Exceptions" external.system.module.type="sourceSet" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager">
    <output url="file://$MODULE_DIR$/../../../../06. Try with resources/What can be released/out/production/classes" />
    <exclude-output />
    <content url="file://$MODULE_DIR$/../../../../06. Try with resources/What can be released/src">
      <sourceFolder url="file://$MODULE_DIR$/../../../../06. Try with resources/What can be released/src" isTestSource="false" />
    </content>
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
    <orderEntry type="module" module-name="19._Exceptions.util.main" />
  </component>
</module>